src/Symfony/Component/Form/Tests/Resources/TranslationFilesTest.php

```diff
@@ -20,7 +20,11 @@ class TranslationFilesTest extends TestCase
      */
     public function testTranslationFileIsValid($filePath)
     {
-        \PHPUnit_Util_XML::loadfile($filePath, false, false, true);
+        if (class_exists('PHPUnit\Util\XML')) {
```

Comment:
to prevent any issue with the BC layer, we should check for the underscore version here